app/models/peoplefinder/concerns/completion.rb

```diff
@@ -21,7 +15,7 @@ def completion_score_fields
       lambda {
         where(%{
           COALESCE(image,'') = ''
-          OR COALESCE(location,'') = ''
+          OR COALESCE(location_in_building,'') = ''
```

Comment:
I know, this is used for getting a list of people who to send a reminder e-mail to - for them to complete the profile. So it should be inline with the completeness and if somebody doesn't have a city filled in (which is counted against completion score), this scope should also retrieve such person.